packages/app/src/sandbox/eval/utils/resolve-utils.js

```diff
@@ -1,6 +1,5 @@
-export function packageFilter(p: Object) {
-  if (!p.main && p.module) {
-    // eslint-disable-next-line
+export function packageFilter(p) {
+  if (p.module) {
```

Comment:
@CompuIves I guess we should also account for `browser` here, maybe something like

```diff
-   if (p.module)
+   if(!p.browser && p.module)
```

Reply:
Hmm yes, I think the resolver prefers browser always over the main/module.